_dev/src/components/configuration/messenger-chat-deprecated.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <b-alert
+    :show="canDisplay"
+    variant="warning"
+  >
+    <div
+      class="d-flex flex-column flex-md-row justify-content-between"
+    >
+      <p class="mb-0">
+        <strong class="ps_gs-fz-16">
+          {{ $t('configuration.disableMessenger.title') }}
+        </strong>
+        <br>
+        <span>
+          {{ $t('configuration.disableMessenger.description') }}
+        </span>
+      </p>
+      <div
+        class="d-md-flex text-center align-items-end mt-2"
+      >
+        <b-button
+          class="mx-1 mt-3 mt-md-0 mr-md-1 text-nowrap ml-auto black-color"
+          variant="link"
+          @click="redirectToFb()"
+        >
+          {{ $t('configuration.disableMessenger.more') }}
+        </b-button>
+        <b-button
+          class="mx-1 mt-3 mt-md-0 mr-md-1 text-nowrap ml-auto"
+          variant="outline-primary"
+          @click="disableMessenger()"
+        >
+          <div class="d-inline-flex">
```

Comment:
I think it's possible to delete this div, it's not recommended to put a div in a button. At worst you can put a span